src/IECoreHoudini/LiveScene.cpp

```diff
@@ -642,24 +642,35 @@ void LiveScene::writeTags( const NameList &tags )
 
 SceneInterface::NameList LiveScene::setNames( bool includeDescendantSets ) const
 {
-	return SceneInterface::NameList();
+	SetCollector allSets;
```

Comment:
The performance of this is terrible. I have to recurse the scene once here again in readSet and probably again in the yet to be implemented hashSet.